docs/feature-gates.md

```diff
@@ -0,0 +1,83 @@
+# Antrea Feature Gates
+
+This page contains an overview of the various feature gates an administrator can
+specify for Antrea components. We follow the same convention as the [Kubernetes
+feature
+gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
+
+In particular:
+ * a feature in the Alpha stage will be disabled by default but can be enabled
+ by editing the appropriate `.conf` entry in the Antrea manifest.
+ * a feature in the Beta stage will be enabled by default but can be disabled
+ by editing the appropriate `.conf` entry in the Antrea manifest.
+ * a feature in the GA stage will be enabled by default and cannot be disabled.
+
+Some feature gates are specific to the Agent, others are specific to the
+Controller, and some apply to both and should be enabled / disabled consistently
+in both `.conf` entries.
+
+To enable / disable a feature, edit the Antrea manifest appropriately. For
+example, to enable `AntreaProxy` on Linux, edit the Agent configuration in the
+`antrea` ConfigMap as follows:
+```yaml
+  antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+    featureGates:
+    # Enable antrea proxy which provides ServiceLB for in-cluster services in antrea agent.
+    # It should be enabled on Windows, otherwise NetworkPolicy will not take effect on
+    # Service traffic.
+      AntreaProxy: true
+```
+
+## Stages for all Feature Gates
```

Comment:
Replace with 'Feature gates for all experimental features'?

A general question: In the doc, I felt "feature gates" was used as a substitute for "features" at some places. Will it be more clear to mention features explicitly as feature gates is a map that store feature names?

Reply:
Did not realize that change is already merged :) If you agree with comment, please consider a follow-up change.

Reply:
Will update later. Although I prefer "List of Available Feature Gates"